packages/roosterjs-content-model-core/lib/editor/core/DOMHelperImpl.ts

```diff
@@ -81,6 +81,47 @@ class DOMHelperImpl implements DOMHelper {
         const paddingRight = parseValueWithUnit(style?.paddingRight);
         return this.contentDiv.clientWidth - (paddingLeft + paddingRight);
     }
+
+    /**
+     * Wrap a node with a wrapper element
+     * @param node
+     * @param wrapper
+     * @returns
+     */
+    wrap(node: Node, wrapper: keyof HTMLElementTagNameMap | HTMLElement): HTMLElement {
```

Comment:
we already have wrap and unwrap API in roosterjs-content-model-dom.

DOMHelper is only used for the case that need to access root content DIV. For other cases, it can be a standalone API